ctk/idp/coverage/Core.md

```diff
@@ -120,8 +120,8 @@ Unmarked sections need attention
 -	5.1 Signing Assertions
 -	5.2 Request/Response Signing
 -	5.3 Signature Inheritance
-	5.4 XML Signature Profile
-		5.4.1 Signing Formats and Algorithms
++	5.4 XML Signature Profile
++		5.4.1 Signing Formats and Algorithms
 +		5.4.2 References
 -		5.4.3 Canonicalization Method
 		5.4.4 Transforms
```

Comment:
If we're saying all of 5.4 is covered, we should have pluses or minuses on all 5.4.x sections too.